src/yang/example-compb-diffserv-filter-policy.yang

```diff
@@ -106,20 +69,20 @@ module example-compb-diffserv-filter-policy {
       leaf loss-priority {
         type enumeration {
           enum high {
-           description "High Loss Priority";
+           description "High Loss Priority.";
```

Comment:
s/Loss Priority/loss priority/

unless you are referring to a special kind of high loss priority.